turbopack/crates/turbo-tasks/src/task/local_task.rs

```diff
@@ -126,31 +99,6 @@ impl fmt::Display for LocalTaskType {
 }
 
 impl LocalTaskType {
-    /// Returns the name of the function in the code. Trait methods are
-    /// formatted as `TraitName::method_name`.
-    ///
-    /// Equivalent to [`ToString::to_string`], but potentially more efficient as
-    /// it can return a `&'static str` in many cases.
-    pub fn get_name(&self) -> Cow<'static, str> {
-        match self {
-            Self::Native {
-                native_fn,
-                this: _,
-                arg: _,
-            } => Cow::Borrowed(native_fn.name),
-            Self::ResolveNative {
-                native_fn,
-                this: _,
-                arg: _,
-            } => format!("*{}", native_fn.name).into(),
-            Self::ResolveTrait {
-                trait_method,
-                this: _,
-                arg: _,
-            } => format!("*{}::{}", trait_method.trait_name, trait_method.method_name).into(),
-        }
-    }
-
```

Comment:
`get_name` would still be more efficient than going through the `Display` trait for `LocalTaskType::Native`, but I guess nothing is calling it?